### ISSUE TITLE
about the uncertainty loss

### ISSUE THREAD

**Issue by J-Wu97** [lorenmt/mtan#51]
你好，当我使用您代码中的uncertainty loss进行训练的时候
loss = sum(1 / (2 * torch.exp(logsigma[i])) * train_loss[i] + logsigma[i] / 2 for i in range(3))
整个loss在经过几个epoch之后会一直是负的，请问您训练时是这样的情况吗？
此外，想问下您为什么logsigma全部初始化为-0.5。
谢谢！

**Comment by J-Wu97** [comment]
> 我建议print per-task performance 或者乘以weighting 前的 training loss 就可

谢谢你的建议！
